src/main/tables/82_job_tickets.sql

```diff
@@ -0,0 +1,18 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records iRODS tickets that were generated for DE jobs.
+--
+CREATE TABLE job_tickets(
+   job_id uuid NOT NULL,
+   ticket varchar(16) NOT NULL,
```

Comment:
This looks like too short a field to me -- the tickets we generate are usually UUIDs

Reply:
Oh, okay. I based the field length on the tickets generated by `iticket`. Thanks for catching that.

Reply:
It looks like the ICAT itself uses a `varchar(100)`, if you just want to copy that!

Reply:
Will do.